explainer.md

```diff
@@ -1,22 +1,26 @@
 ## Explainer
 
-Author: Eric Willigers &lt;<ericwilligers@chromium.org>&gt;<br>
-Author: Jay Harris &lt;<harrisjay@chromium.org>&gt;<br>
-Author: Raymes Khoury &lt;<raymes@chromium.org>&gt;
+Authors: 
+* Eric Willigers &lt;<ericwilligers@chromium.org>&gt;<br>
+* Jay Harris &lt;<harrisjay@chromium.org>&gt;<br>
+* Darwin Huang &lt;<huangdarwin@chromium.org>&gt;<br>
+* Raymes Khoury &lt;<raymes@chromium.org>&gt;
```

Comment:
A la the Font Access explainer, I think you should update this to reflect that you are the current owner for FH and the others listed here are emeritus.

Reply:
Sounds good. I've moved other editors to a "Former editors" section, like the Font Access explainer.